lib/App.pm

```diff
@@ -45,6 +45,11 @@ sub get_path {
 
     # Default personal clamtk directory
     $path->{ clamtk } = $path->{ directory } . '/.clamtk';
```

Comment:
Why do we still keep this line of assignment while re-assign the value in the next line?

Reply:
Yeah, that looks like a mistake. That'll be in 6.18 which can probably be pushed Friday.